packages/transit-vehicle-overlay/src/components/markers/ModeRectangles/images/bus.js

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+
+export default function Bus(props) {
+  return (
+    <svg
```

Comment:
At some point in the future, we may need to watch for duplication with the icons package.